_posts/2017-08-01-emldown.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "emldown: From machine readable EML metadata to a pretty documentation website"
+author: "Maëlle Salmon, Andrew MacDonald, Kara Woo, Carl Boettiger, Jeff Hollister"
+date: "2017-08-01"
+output:
+  md_document:
+    variant: markdown_github
+    toc: true
+---
+
+We all know we should document our data, and some documentation is better than no documentation.  Even better, it's best practice to write metadata for a dataset based on a standard metadata format than in an unstructured document. A well-established standard is the [Ecological Metadata Language (EML)](https://knb.ecoinformatics.org/#external//emlparser/docs/index.html). If you follow the standard properly, you're sure to have documented your data well, which gives you a good feeling. Moreover, this will allow easier sharing of that dataset later. 
```

Comment:
I see a bit of metadata definition below so perhaps review the order of sentences in the intro